pkg/storage/mvcc_value.go

```diff
@@ -131,15 +133,37 @@ func (v MVCCValue) String() string {
 // SafeFormat implements the redact.SafeFormatter interface.
 func (v MVCCValue) SafeFormat(w redact.SafePrinter, _ rune) {
 	if v.MVCCValueHeader != (enginepb.MVCCValueHeader{}) {
+		fields := make([]string,0)
 		w.Printf("{")
 		if !v.LocalTimestamp.IsEmpty() {
-			w.Printf("localTs=%s", v.LocalTimestamp)
+			fields = append(fields,fmt.Sprintf("localTs=%s", v.LocalTimestamp))
 		}
+		if v.ImportEpoch != 0 {
+			fields = append (fields,fmt.Sprintf("importEpoch=%v",v.ImportEpoch))
+		}
+		w.Print(strings.Join(fields,", "))
 		w.Printf("}")
 	}
 	w.Print(v.Value.PrettyPrint())
 }
 
+// EncodeForExport strips fields from the MVCCValueHeader that
+// should not get exported out of the cluster.
+//
+//gcassert:inline
+func EncodeForExport(mvccValue MVCCValue) ([]byte, error) {
```

Comment:
nit: since we moved this to a top-level function rather than a method, let's do `EncodeMVCCValueForExport()`.